.github/workflows/integration-test-object-suite.yaml

```diff
@@ -0,0 +1,65 @@
+name: Integration test CephObjectSuite
```

Comment:
The new test suite also needs to be added to `integration-tests-on-release.yaml`

Reply:
Right! Also added this to `.mergify.yaml` for merges into master.